WPLGSS/Services/SequenceRunner.cs

```diff
@@ -55,12 +88,31 @@ private void OnTimedEvent(object source, System.Timers.ElapsedEventArgs e)
         // Raises Timed event every millisecond
         public void RunSequence(Sequence sequence)
         {
-            primarySequence = sequence.PrimarySequence;
-            abortSequence = sequence.AbortSequence;
+            this.primarySequence = sequence.PrimarySequence;
+            this.abortSequence = sequence.AbortSequence;
+            SequenceRunningStateChanged(this, new StatusChangedEventArgs());
             timer = new System.Timers.Timer();
             timer.Interval = 1;
-            timer.Elapsed += OnTimedEvent;       
+            timer.Elapsed += OnTimedEvent;
             timer.Enabled = true;
+            sequenceStartTime = DateTime.Now.TimeOfDay;
+            Service.ChannelValueUpdated += Service_ChannelValueUpdated;
+        }
+
+        private void Service_ChannelValueUpdated(object sender, ChannelValueUpdatedEventArgs e)
+        {
+            
+            if(AbortCondition.ThresholdMax <= e.Value || AbortCondition.ThresholdMin >= e.Value)
```

Comment:
We need to support multiple abort conditions. I suggest using a list and checking if any of the conditions are violated.